src/components/RehydrationProvider.js

```diff
@@ -39,16 +39,31 @@ const transformRemoveFetchErr = createTransform(state => omit(state, ['isFetchin
   whitelist: ['session']
 })
 
+// Store only definitions from ui reducer
+const transformUiDefinitions = createTransform(
```

Comment:
do we need to do this transform if we have a separate navigationReducer now?

Reply:
Yes. It is required to align data for everyone that has already used the persistance of data.
If `redux-persist` will find the `ui` property in the storage, it will use that on a first attempt instead of the real reducer.
Probably, the stored copy will contain also the navigation property, which will never be cleaned.
With this function we will clean that stale information, and also store only necessary data.

Reply:
Ahhh that makes sense. Thanks